Add latency SLOs to SLO template

minaslater opened the pull request:
An attempt to adjust the SLO service template to include request latency SLOs (95/99).

minaslater committed:
add latency slos to slo template

OlamideOl1 commented:
This looks good to me, you may need to bump up the helm chart version as well to fix the lint error - https://github.com/thoughtbot/helm-charts/blob/6c8786a6e92b66976219e4b3ad317a1774ad8acd/charts/helm-rails/Chart.yaml#L12 Once you do that, you can reference the new helm chart version in the Github workflow file.

minaslater committed:
Update Helm Rails to version 0.1.9

minaslater committed:
update ubuntu version

jferris commented on charts/helm-rails/templates/_prometheusservicelevel.tpl:
```diff
@@ -25,6 +25,40 @@ spec:
       ticketAlert:
         disable: true
+  - name: "{{ $.Release.Name }}-{{ .namespace.name }}-requests-99-latency"
+    objective: 99
```
Should we use variables for the objective?

jferris commented on charts/helm-rails/templates/_prometheusservicelevel.tpl:
```diff
@@ -25,6 +25,40 @@ spec:
+    description: "Common SLO based on latency for HTTP request responses"
+    sli:
+      events:
+        errorQuery: sum_over_time((sum(rate(istio_request_duration_milliseconds_count{destination_service_name={{ include "fullname" . | quote }}}[5m])))[{{.window}}:5m]) - sum_over_time((sum(rate(istio_request_duration_milliseconds_bucket{destination_service_name={{ include "fullname" . | quote }},le="1500"}[5m])))[{{.window}}:5m])
```
Also could see a variables for this 1500 value.

minaslater committed:
Use variables for SLO definitions

In order to make SLO definitions more flexible, adjusted
prometheusserviceleve.tpl to take on variables from values.yaml to
define the details of the SLO objectives.
